powerauth-java-server/src/test/java/io/getlime/security/powerauth/app/server/controller/api/PowerAuthControllerTestConfig.java

```diff
@@ -0,0 +1,493 @@
+    }
+
+    public String getApplicationKey() {
+        return applicationKey;
+    }
+
+    public void setApplicationKey(final String applicationKey) {
+        this.applicationKey = applicationKey;
+    }
+
+    public String getApplicationSecret() {
+        return applicationSecret;
+    }
+
+    public void setApplicationSecret(final String applicationSecret) {
+        this.applicationSecret = applicationSecret;
+    }
+
+    public String getMasterPublicKey() {
+        return masterPublicKey;
+    }
+
+    public void setMasterPublicKey(final String masterPublicKey) {
+        this.masterPublicKey = masterPublicKey;
+    }
+
+    public String getApplicationVersion() {
+        return applicationVersion;
+    }
+
+    public void setApplicationVersion(final String applicationVersion) {
+        this.applicationVersion = applicationVersion;
+    }
+
+    public String getApplicationName() {
+        return applicationName;
+    }
+
+    public Long getLoginOperationTemplateId() {
+        return loginOperationTemplateId;
+    }
+
+    public void setLoginOperationTemplateId(final Long loginOperationTemplateId) {
+        this.loginOperationTemplateId = loginOperationTemplateId;
+    }
+
+    public String getLoginOperationTemplateName() {
+        return loginOperationTemplateName;
+    }
+
+    public void setLoginOperationTemplateName(final String loginOperationTemplateName) {
+        this.loginOperationTemplateName = loginOperationTemplateName;
+    }
+
+    public String getActivationId() {
+        return activationId;
+    }
+
+    public void setActivationId(final String activationId) {
+        this.activationId = activationId;
+    }
+
+    public void setActivationCode(final String activationCode) {
+        this.activationCode = activationCode;
+    }
+
+    public String getActivationCode() {
+        return activationCode;
+    }
+
+    public void setActivationName(final String activationName) {
+        this.activationName = activationName;
+    }
+
+    public String getActivationName() {
+        return activationName;
+    }
+
+    /**
+     * Creates and configures a new {@link PowerAuthClient} bean.
+     * <p>
+     * The method configures and returns a PowerAuthClient instance for interacting with
+     * the PowerAuth Server. It sets up the client with the necessary configurations such as
+     * accepting invalid SSL certificates for testing purposes.
+     *
+     * @return A configured instance of PowerAuthClient
+     * @throws Exception if there is an issue creating the PowerAuthClient instance
+     */
+    @Bean
+    public PowerAuthClient powerAuthClient() throws Exception {
+        final PowerAuthRestClientConfiguration config = new PowerAuthRestClientConfiguration();
+        config.setAcceptInvalidSslCertificate(true);
+        return new PowerAuthRestClient(POWERAUTH_REST_URL);
+    }
+
+
+    /*
+     *****************************************************************
```

Comment:
Can we avoid this style of comments with many `*` characters? It's not common in Java.